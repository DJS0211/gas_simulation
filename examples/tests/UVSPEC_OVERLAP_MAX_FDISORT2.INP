data_files_path ../../data/
atmosphere_file ../../data/atmmod/afglus.dat
source solar ../../data/solar_flux/atlas_plus_modtran
mol_modify O3 300. DU
day_of_year 170          # Correct for Earth-Sun distance
albedo 0.2               # Surface albedo
sza 32.0                 # Solar zenith angle
number_of_streams  6                  # Number of streams

rte_solver fdisort2      # This overrides what is specified in above file
wavelength 544.7 544.7   # Wavelengths considered

wc_file 1D ../WC.DAT  # Location of the water cloud file
wc_properties hu

ic_file 1D ../IC.DAT  # Location of the ice cloud file
ic_properties fu

cloud_fraction_file  ../CF.DAT
cloud_overlap  max
output_user lambda edir edn eup uavgdir uavgdn uavgup tcc

mol_abs_param crs

quiet
