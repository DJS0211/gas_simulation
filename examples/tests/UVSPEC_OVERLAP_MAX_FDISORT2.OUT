  544.700  2.639512e+02  5.859536e+02  1.699810e+02  2.476994e+01  8.444264e+01  2.705332e+01  8.000000e-01 
